tests/e2e/specs/AttributionDetailsStep.js

```diff
@@ -0,0 +1,43 @@
+module.exports = {
+    '@tags': ['att'],
+    'AttributionDetailsStep'(browser){
+        const knowLicenseSelector = '.b-radio'
```

Comment:
This may be a bit much, a lot of things being checked are static and always rendered since they're right there in the markdown, but I think it's ok for now